lib/open_rtb_ecto/v2/bid_request/video.ex

```diff
@@ -100,14 +100,13 @@ defmodule OpenRtbEcto.V2.BidRequest.Video do
     |> validate_required([:mimes])
     |> validate_inclusion(:linearity, 1..2)
     |> validate_inclusion(:protocol, 1..10)
-    |> validate_subset(:protocols, 1..10)
```

Comment:
Did you mean to totally remove this or just increase the max?

Reply:
thought it might be better to remove it and just assert that the value is a integer, rather than rejecting the whole request . Maybe we can repeat this elsewhere and be less strict with these values.

Reply:
yeah we spoke in slack, I think removing it is fine. We will only ever have ints in here thanks to the casting, and making sure we keep the possible values updated is probably unnecessary